test/functional/fortran/c_interface_module.F90

```diff
@@ -46,6 +46,7 @@ module C_interface_module
     ! include support for more character types.
     !
 
+#ifndef Spglib_HAS_C_UNSIGNED
     ! Fortran does not (yet) support unsigned types.
     integer, parameter :: &
         C_unsigned = C_int, &
```

Comment:
Wait, this is only used in the test interface? Can you investigate if these can be dropped instead?

Reply:
Yeah, the variables are actually unnecessary.